### PR TITLE
introduce metrics for software stacks maintained by Kebechet

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/metrics-exporter/issues/791

### CODE REVIEW

File: thoth/metrics_exporter/metrics.py [hunk 1]
```diff
@@ -251,3 +251,7 @@
     "thoth_kebechet_current_rate_limit",
     "Kebechet remaining rate limit.",
 )
+
+kebechet_maintained_software_stacks_count = Gauge(
+    "thoth_kebechet_maintained_software_stacks_count", "Count of number of software stacks maintained by Kebechet.", []
```
Comment (fridex) [inline thoth/metrics_exporter/metrics.py:256]:
```suggestion
    "thoth_kebechet_maintained_software_stacks_count", "Count the number of software stacks maintained by Kebechet.", []
```

**Review by fridex — COMMENTED** [review]
One minor typo.

File: thoth/metrics_exporter/metrics.py (hunk 1 above)
Comment (pacospace) [reply thoth/metrics_exporter/metrics.py]: Thanks @fridex, updated!

**Review by fridex — APPROVED** [review]
lgtm 👍🏻

**Pull request merged**